utide/confidence.py

```diff
@@ -134,7 +134,8 @@ def nearestSPD(A):
         # Tweaking strategy differs from D'Errico version.  It
         # is still a very small adjustment, but much larger than
         # his.
-        maxeig = np.linalg.eigvals(Ahat).max()
+        # Eigvals are or can be complex dtype, so take abs().
```

Comment:
This is the fix for the problem that arose on Travis but not on my machine.  On Travis, without this change, the subsequent `np.spacing(maxeig)` raised an exception (TypeError, I think) because it could not cast complex128 to an acceptable dtype.